python-backend/app/api/mines/mine/resources/mine.py

```diff
@@ -132,7 +132,14 @@ def mine_status_processor(self, mine_status, mine_guid):
         except AssertionError as e:
             self.raise_error(400, 'Error: {}'.format(e))
         mine_status_xref.save()
+
         try:
+            existing_status = MineStatus.find_by_mine_guid(mine_guid)
+            if existing_status:
+                existing_status.mine_status_xref_guid = mine_status_xref.mine_status_xref_guid
+                existing_status.save()
+                return existing_status
```

Comment:
So, we're doing this to avoid creating new `mine_status` records on a `PUT`? I'm guessing for the benefit of update-in-place giving the freedom to not have to expire records?

My only concern is if we're leaving orphaned `MineStatusXref` records. I haven't worked with mine status before, so I could be missing something, but it looks like that's what's happening here.

Reply:
So far as I can tell MineStatusXref is actually the lookup table, which has permutations of code, reason_code, and sub_reason_code. It doesn't relate to mine_status and isn't actually a xref, so nothing will get orphaned.

Reply:
Alrighty. Sounds like you've looked at this further than I have, so I'll defer to you. Marking as resolved